src/index.html

```diff
@@ -5,8 +5,295 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <header class="header">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="moyo-logo"
+            class="header__img"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">apple</a>
+            </li>
+            <li class="nav__item">
```

Comment:
Add empty lines between multiline siblings.
```suggestion
            </li>
            
            <li class="nav__item">
```